src/cdomains/offset_intf.ml

```diff
@@ -0,0 +1,113 @@
+type 'i t = [
+  | `NoOffset
+  | `Field of CilType.Fieldinfo.t * 'i t
+  | `Index of 'i * 'i t
+] [@@deriving eq, ord, hash]
+
+(* TODO: remove? *)
+type 'i offs = 'i t [@@deriving eq, ord, hash]
```

Comment:
What is the plan with this TODO? Do we want to do it or not?

Reply:
At some point I thought it could be removed but actually it cannot be. Some inner signatures define their own `type t` but need to refer to this outer `t`, so `offs` is the alias necessary to do that.
I've replaced the TODO with this note.